src/app/static/src/app/components/modelCalibrate/ModelCalibrate.vue

```diff
@@ -85,7 +87,15 @@
             ...mapStateProps<ModelCalibrateState, keyof Computed>(namespace, {
                 loading: s => s.fetching,
                 calibrating: s => s.calibrating,
-                error: s => s.error
+                error: s => s.error,
+                progressMessage: s => {
+                    if (s.status && s.status.progress && s.status.progress.length > 0) {
```

Comment:
`status.progress` is an array but in practice it currently only ever contains one item